io.gitlab.hamadmarri.gamma.json

```diff
@@ -0,0 +1,62 @@
+{
+	"app-id": "io.gitlab.hamadmarri.gamma",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.34",
+	"sdk": "org.gnome.Sdk",
+	"command": "gamma",
+	"finish-args": [
+		"--share=network",
```

Comment:
What is network access for?

Reply:
I heard that gtk require networking, also, in future the text editor will download and install plugins